spec/models/travel_request_spec.rb

```diff
@@ -14,4 +15,18 @@
     it { is_expected.to respond_to :notes }
     it { is_expected.to respond_to :estimates }
   end
+
+  # this acts as a "master test" for all request types using this property
+  describe "status enum" do
+    subject(:travel_request) { described_class.create(creator: creator) }
+    let(:creator) { FactoryBot.create(:staff_profile) }
+    # test this here for now; it won't ever come through form params.
```

Comment:
Is this comment for the argument error?

Reply:
@carolyncole I updated the comment; is that more clear? Or should I just take it out?